packages/core/src/Services/logout.js

```diff
@@ -12,6 +12,7 @@ const doLogout = response => {
     if (response.logout !== 1) return undefined;
     removeCookies('affiliate_token', 'affiliate_tracking', 'onfido_token');
     SocketCache.clear();
+    localStorage.removeItem('launchModalShown');
```

Comment:
Should it be cleared with each logout? If that's the case, you may wanna use sessionStorage instead.

Reply:
thanks